docs/servo_power_test/src/operate_port.ts

```diff
@@ -25,14 +25,17 @@ export class OperatePort {
     if (this.port === undefined) return;
     await this.port.open({baudRate: 115200});
   }
-  public async close() {
-    if (this.port === undefined) return;
+  public async readCancel() {
```

Comment:
Maybe it's a good idea to write a function-level documentation to explain the difference between readCancel and close (on top of each function).